columntypes/columntypes.go

```diff
@@ -116,6 +116,8 @@ func GetColumnTypes(
 	var columnName string
 
 	rows, err := repo.DB.QueryxContext(ctx, fmt.Sprintf(queryColumnDescription, tableName))
+	defer rows.Close() //nolint:staticcheck // error doesn't affect outcome
```

Comment:
We shouldn't close the rows before we check the error, rows might be `nil` (see [example](https://pkg.go.dev/database/sql#example-Rows)).